tests/integration/ceph_mgr_test.go

```diff
@@ -205,6 +206,31 @@ func (s *CephMgrSuite) waitForOrchestrationModule() {
 	}
 	require.Nil(s.T(), err)
 }
+
+func (s *CephMgrSuite) CompareNodes(orchestratorNodes []string) error {
+	// get the k8s nodes
+	nodes, err := k8sutil.GetNodeHostNames(s.k8sh.Clientset)
+	if err == nil {
+		if len(nodes) == 0 {
+			return fmt.Errorf("k8s cluster does not return any cluster nodes")
+		} else {
+			k8sNodes := make([]string, 0, len(nodes))
+			for k := range nodes {
+				k8sNodes = append(k8sNodes, k)
+			}
+			sort.Strings(k8sNodes)
+			sort.Strings(orchestratorNodes)
+			logger.Infof("Comparing nodes: kubernetes nodes = %s // orchestrator nodes = %s", k8sNodes, orchestratorNodes)
+			if assert.Equal(s.T(), orchestratorNodes, k8sNodes){
```

Comment:
If the test is going to retry, it shouldn't have a test assert before the retries are done. This will fail the test, right?

Reply:
learned!